my-app/src/App.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import "./App.css";
+import { MembersCardsSection } from "./components/membersCardsSection/MembersCardsSection";
+import { Button } from "./components/buttons/button/Button";
+
+function App() {
+  return (
+    <div className="App">
+      <div className="container">
+        <p>Add members to Front-end development team</p>
+        <MembersCardsSection showSearch={true}/>
+        <div className="buttons">
+        <Button  variant="cancel" />
```

Comment:
You can pass a text prop to avoid relying on the variant to set the default button text. This reduces dependency on internal logic.

Something like this:

`<Button variant="cancel" text="Cancel" />`

Instead of fully depending on the variant

Reply:
Done